src/shared/filesystemUtilities.ts

```diff
@@ -9,10 +9,15 @@ import * as fs from 'fs'
 import * as os from 'os'
 import * as path from 'path'
 import { promisify } from 'util'
-import { access, PathLike, readdir, readFile } from './filesystem'
+import { access, mkdir, PathLike, readdir, readFile } from './filesystem'
 
 const DEFAULT_ENCODING: BufferEncoding = 'utf8'
 
+export const tempDirPath = path.join(
```

Comment:
Can you create a tracking issue with a link to the upstream docker permissions issue, then add a comment with a link to the tracking issue? This will allow us to remove the workaround if/when the upstream issue is fixed.

Reply:
✔️